assets/js/banCtrl.js

```diff
@@ -33,28 +33,33 @@ define([
                 return;
             }
 
-            if ($scope.banInfo.banNote.length > 300) {
-                $scope.banError = "The ban note cannot be longer than 300 characters.";
+            if ($scope.banInfo.username.substring(0,3) === '/u/') {
```

Comment:
Oh, quick thing, this doesnt' account for "u/" on it's own. Should we allow that? Or just assume mods aren't silly enough to do it?